xknx/io/tunnel.py

```diff
@@ -236,6 +238,16 @@ async def _disconnect_request(self, ignore_error: bool = False) -> None:
             )
         self.communication_channel = None
 
+    async def request_description(self) -> GatewayDescriptor | None:
+        """Request description from tunneling server."""
+        description = RequestDescription(
```

Comment:
Same as above, the naming is kinda off, not sure how you would start a description :D.